lib/Twig/Extension.php

```diff
@@ -76,4 +76,12 @@ public function getGlobals()
     {
         return array();
     }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function getName()
+    {
```

Comment:
Well, is Twig really calling this method for a class *not* overriding it ? I don't think so. and classes overriding it would *not* trigger a deprecation there